server/src/http_google_analytics.go

```diff
@@ -40,7 +40,7 @@ func httpGoogleAnalytics(c *gin.Context) {
 	if cookie, err := r.Cookie("_ga"); err != nil {
 		// They don't have a cookie set, so set a new one
 		clientID = uuid.NewV4().String()
-		http.SetCookie(w, &http.Cookie{ // nolint: exhaustivestruct
+		http.SetCookie(c.Writer, &http.Cookie{ // nolint: exhaustivestruct
```

Comment:
looks like there are many changes in this PR that have nothing to do with refactoring a logger package

Reply:
Same as above

Reply:
instead of changing code, just set the linter version to one from 2019